### PR TITLE
[Keyboard] Add Acid and Co. Stellaron TKL

### PR DESCRIPTION
**Pull request by itsme-zeix**
## Description

Added support for Acid and Co's Stellaron TKL.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/zeix/acidandco/stellaron/keyboard.json [hunk 1]
```diff
@@ -0,0 +1,482 @@
+{
+    "manufacturer": "Acid & Co",
+    "keyboard_name": "Stellaron TKL",
+    "maintainer": "itsme-zeix",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "indicators": {
+        "caps_lock": "GP27",
+        "on_state": 1,
```
Comment (fauxpark) [inline keyboards/zeix/acidandco/stellaron/keyboard.json:9]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/zeix/acidandco/stellaron/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+[0] = LAYOUT_all(
+  KC_ESC,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_F13,           KC_PSCR, KC_SCRL, KC_PAUS,
+  KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,          KC_INS,  KC_HOME, KC_PGUP,
+  KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,          KC_DEL,  KC_END,  KC_PGDN,
+  KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,           
+  KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT, MO(1),                     KC_UP, 
+  KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             KC_RALT, KC_RGUI, MO(1),   KC_RCTL,          KC_LEFT, KC_DOWN, KC_RIGHT)
```
Comment (zvecr) [inline keyboards/zeix/acidandco/stellaron/keymaps/default/keymap.c:26]: `MO(1)` used here would go out of bounds, and either needs to be replaced or a 2nd layer added.

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/zeix/acidandco/stellaron/keyboard.json (hunk 1 above)
Comment (itsme-zeix) [reply keyboards/zeix/acidandco/stellaron/keyboard.json:9]: The hardware configuration has the reverse behaviour for this (tested and confirmed on PCB).

Comment (fauxpark) [reply keyboards/zeix/acidandco/stellaron/keyboard.json:9]: `1` is the default, it can be safely removed.

Comment (itsme-zeix) [reply keyboards/zeix/acidandco/stellaron/keyboard.json]: Gotcha! Have made the change.

**Comment by itsme-zeix** [comment]
Bump for review!

**Review by fauxpark — APPROVED** [review]

**Comment by itsme-zeix** [comment]
Removed `./keymaps/via` in compliance with the August 26, 2024 update to transition VIA-enabled keymaps to the VIA QMK Userspace repo.

**Pull request merged**